### ISSUE TITLE
SpaceInsideHashLiteralBraces should allow comment after opening brace

### ISSUE THREAD

**Issue by sferik** [rubocop/rubocop#957]
In https://github.com/bbatsov/rubocop/issues/944#issuecomment-39534040, I mentioned that the `SpaceBeforeFirstArg` cop was generating a false positive offense for the following code:

``` ruby
ActionMailer::Base.smtp_settings = {
  address: 'smtp.sendgrid.net',
  port: '25',
  authentication: :plain,
  user_name: ENV['SENDGRID_USERNAME'],
  password: ENV['SENDGRID_PASSWORD'],
  domain: ENV['SENDGRID_DOMAIN'],
}
```

I tried disabling that offense as follows:

``` ruby
ActionMailer::Base.smtp_settings = { # rubocop:disable SpaceBeforeFirstArg
```

This generated a new offense from the `SpaceInsideHashLiteralBraces` cop:

```
C: Space inside { detected.
```

It seems to me that the `SpaceInsideHashLiteralBraces` cop should allow a comment (with a space before it) after opening brace.
